src/ts/components/subtitleoverlay.ts

```diff
@@ -92,11 +96,8 @@ export class SubtitleOverlay extends Container<ContainerConfig> {
       this.updateComponents();
     };
 
-    player.on(player.exports.PlayerEvent.AudioChanged, subtitleClearHandler);
```

Comment:
Looking at your commits it looks like the player is firing `PlayerEvent.CueExit` with these removed events? The UI is defined to be backwards compatible with all player releases of a major version - so this works correctly back to 8.0.0.

Reply:
Right, the intention to remove these callbacks was that the cues get cleaned up by the `PlayerEvent.CueExit` rather than doing it on these actions. This lead to a cue that spans over a longer time range would be cleared although it should still be visible.

Reply:
> This lead to a cue that spans over a longer time range would be cleared although it should still be visible.

You mean when seeking within the queue's timespan? Then this case could specifically be handled. The reason why the callbacks were there was that the cues did not get cleaned up.

Reply:
I thought so. I'll see if I can come up with an improved cleanup strategy.

Reply:
This is still an open point to resolve.